prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -304,11 +316,11 @@ class ReportFunction {
     private fun createdResponse(
         request: HttpRequestMessage<String?>,
         validatedRequest: ValidatedRequest,
-        destinations: List<String>
+        responseBody: String,
```

Comment:
I refactored the creation of the body up one level, so the ActionHistory could get to it.